features/node/seccomp.feature

```diff
@@ -16,3 +16,43 @@ Feature: Seccomp
     And the output should not contain:
       | 2 |
 
+  # @author jhou@redhat.com
+  # @case_id OCP-32065
+  @admin
+  Scenario: Using Secure Computing Profiles with Pod Annotations
+    # Create custom machine config that contains the seccomp
+    Given I switch to cluster admin pseudo user
+    And I obtain test data file "node/machineconfig_nostat.yaml"
+    When I run the :create admin command with:
+       | f | machineconfig_nostat.yaml |
+    Then the step should succeed
+    And admin ensures "custom-seccomp" machineconfig is deleted after scenario
+
+    # Wait for machineconfigpool to process the new config
+    Given I wait up to 600 seconds for the steps to pass:
+    """
+    Given evaluation of `machine_config_pool('worker').get` is stored in the :mcp_worker clipboard
+    Given evaluation of `machine_config_pool('worker').condition(type: 'Updating')` is stored in the :mcp clipboard
```

Comment:
Since `machine_config_pool('worker').condition(type: 'Updating')` is cached data, I had to use `machine_config_pool('worker').get` in previous step to refresh it. Please let me know if you have more elegant ways.

Reply:
Perhaps we can use `machine_config_pool('worker').condition(type: 'Updating', cached: false)`

Reply:
That's cool, didn't know this magic!